src/main/java/com/facebook/bitEncoderDecoder/bitwise/BitwiseEncoderImpl.java

```diff
@@ -70,4 +71,18 @@ private String checkIfEven(String triplet) {
         return triplets;
     }
 
+    private String doubleTriplet(String input) {
+        StringBuilder encodeString = new StringBuilder();
+        if (input == null) {
+            throw new IllegalArgumentException();
```

Comment:
If I understand correctly, this may be in future some InternalError, because null in this place means that something went terribly wrong.